core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -21,7 +21,9 @@
 import io.confluent.common.config.ConfigDef.Importance;
 import io.confluent.common.utils.SystemTime;
 import io.confluent.common.utils.Time;
+import jersey.repackaged.com.google.common.collect.Lists;
```

Comment:
Do we need to rely on this? I'm actually not sure whether Jersey considers this public API? In particular, it looks like an `Arrays.asList` could work fine here? Or is this just because Guava folks know Guava and IntelliJ found the import for you?